{{ cookiecutter.slug }}/renovate.json

```diff
@@ -24,5 +24,19 @@
   "labels": [
     "dependency"
   ],
-  "separateMinorPatch": true
+  "separateMinorPatch": true,
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["patch"],
+      "matchCurrentVersion": "!/^v?0\\./",{# negative match: do not match versions that match regex `^v?0\.` #}
```

Comment:
Did we decide somewhere that `v0` patches are seen as dangerous? I'd rather have a list of components where we don't want auto-merges. Loads of our internal tools are `v0`. Making it configurable at least would be nice.

Reply:
It will become configurable. That's in a separate ticket. This is simply going to be the default.

Reply:
The acceptance criteria clearly say that we don't want to blanket-automerge v0 patch releases because according to semver, they can always be breaking.

Reply:
OK i see. Was not aware. if we already have an if can we also do somthing like `cookiecutter.automerge_patch–v0`?

Reply:
Fair enough. Makes sense to have a global enable-disable for that as well, and then package-specific exceptions later in that other ticket.